WebApiCoreSeed.Data/.vs/config/applicationhost.config

```diff
@@ -0,0 +1,992 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I think this file may be specific to each user... maybe? I'm not fully convinced, but I just saw that there's some data tied to the particular configuration of the developer's machine, like disk paths and so on.